### ISSUE TITLE
Separate `PresentationURL` from presentation request

### ISSUE THREAD

**Issue by awoie** [decentralized-identity/presentation-request#3]
To allow the presentation request and responses to be sent via DIDComm and to avoid duplicate attributes in the messages, we should refactor out the `PresentationURL` from the presentation request.

@tplooker what do you think?

**Comment by rado0x54** [comment]
Hey Oliver, I share this input and feel that the communication endpoints are ideally abstracted behind a DID and resolved via DID comms.

**Comment by awoie** [comment]
Think of DIDComm providing the basic common structure that includes a `reply` address which could be anything (e.g., HTTPS, BLE). This can then be extended using additional properties that the presentation request needs. 

I guess, it might be also possible to use the same name for the `reply` property but this feels redundant.

**Comment by tplooker** [comment]
@awoie I agree some of the information in these examples is more generally information at a messaging level that something like DIDComm could supply like how to reply
